MultiSource/Benchmarks/Olden/bisort/bitonic.c

```diff
@@ -7,13 +7,16 @@
 #include <stdchecked.h>
 #include <stdio_checked.h>
 
+#pragma BOUNDS_CHECKED ON
+
 #define CONST_m1 10000
 #define CONST_b 31415821
 #define RANGE 100
 
 int NumNodes, NDim;
 
 int random(int);
+void *calloc(size_t nmemb, size_t size) : byte_count(nmemb * size);
```

Comment:
Yes. calloc is declared in stdlib_checked.h
 But, in this application, it defines its own  random function but stdlib.h also has  random 
 To avoid redefinition error of  random function, I did not include  stdlib_checked.h